base/utils/R/read.output.R

```diff
@@ -141,6 +141,11 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
       for (v in variables) {
         if (v %in% c(names(nc$var), names(nc$dim))) {
           newresult <- ncdf4::ncvar_get(nc, v)
+          if("pft.numbers" %in% names(nc$dim)){
```

Comment:
Standard dimension name is pft not pft.numbers
https://github.com/PecanProject/pecan/blob/develop/base/utils/data/standard_vars.csv#L5